### PR TITLE
tests refactoring, replace test fixtures, add admin config validation schema

### PR DESCRIPTION
**Pull request by Kuyantsev-VP**

### CODE REVIEW

**Comment by Kuyantsev-VP** [comment]
First  need to merge this PR: https://github.com/rvadim/elastalert-k8s-automation/pull/8

**Comment by Kuyantsev-VP** [comment]
environment variables with passwords added to validating config
added checks for validation of admin config

File: config.py [hunk 1]
```diff
@@ -1,16 +1,35 @@
...
 
+class PropertyNotExistError(ConfigurationError):
+    def __init__(self, prop_name: str, *args):
+        self.message = 'Required property \'' + prop_name + '\' is not specified.'
+        super().__init__(self.message, *args)
+
+
+class PropertyFormatError(ConfigurationError):
+    def __init__(self, prop_name: str, incomming_message: str, *args):
+        self.message = 'Validation of property \'' + \
```
Comment (rvadim) [inline config.py:20]:
```
[flake8]
exclude = .git,deploy
max-line-length = 119
```
Also concatenation not required

**Review by rvadim — APPROVED** [review]
LGTM

**Review by geshido — APPROVED** [review]
LGTM

**Comment by rvadim** [comment]
Please rebase and squash commits

**Pull request merged**